Gemfile.lock

```diff
@@ -195,6 +195,7 @@ GEM
 PLATFORMS
   arm64-darwin-21
   x86_64-darwin-20
+  x86_64-darwin-21
```

Comment:
I am on an Intel mac running the latest macOS version and this was added after I bundled so I assume it's just because I didn't bundle from my m1.